docs_src/microservices/application/Triggers.md

```diff
@@ -27,13 +31,98 @@ Type = 'zero' #specifies of message bus (i.e zero for ZMQ)
         Port = 5563
         Protocol = 'tcp'
 ```
-By default, `EdgeX Core Data` publishes data to the `events`  topic using ZMQ on port 5563. The publish host is used if publishing data back to the message bus. 
+By default, `EdgeX Core Data` publishes data to the `events`  topic using ZMQ on port 5563. The publish host is used if publishing data back to the message bus. As stated above there are three implementations you can choose from. These type values are as follows:
+
+```
+zero - for ZeroMQ
+mqtt - for MQTT (Requires a MQTT Broker running and Core-Data configure to use MQTT)
+redisstreams - for Redis Streams (Requires Redis running and Core-Data configure to use Redis Streams)
+```
 
 !!! important
```

Comment:
Your important and the note below are not coming through as they are not indented beneath the Admonitions